x-pack/plugins/apm/public/components/shared/item_badge/index.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+
+interface Props {
+  items: string[];
+  multipleItemsMessage?: string;
+}
+export function ItemsBadge({
+  items = [],
+  multipleItemsMessage = i18n.translate(
+    'xpack.apm.agentExplorer.serviceNameSelect.placeholder',
+    {
+      values: { itemsCount: items.length },
+      defaultMessage: '{itemsCount, plural, one {1 item} other {# items}}',
+    }
+  ),
+}: Props) {
+  if (items.length < 2) {
+    return (
+      <>
+        {items.map((item) => (
+          <EuiBadge color="hollow" key={item}>
+            {items}
+          </EuiBadge>
+        ))}
+      </>
+    );
+  }
+  return (
+    <EuiToolTip
+      position="right"
+      content={items.map((item) => (
+        <React.Fragment key={item}>
+          {item}
+          <br />
```

Comment:
I know that this was just copied over from `EnvironmentBadge` but `EuiSpacer` could be used here

Reply:
Too much space when I use the `EuiSpacer`
<img width="866" alt="image" src="https://user-images.githubusercontent.com/1313018/197553061-ad41fb65-b295-48b1-bf58-44a1d30994e4.png">

Reply:
Even if you change the size (e.g. `<EuiSpacer size="xs" />`)?